README.md

```diff
@@ -1,16 +1,49 @@
 # Exercises
 
-__NOTE:__ DO NOT USE THE BUILT IN METHODS WHEN ASKED TO MAKE THEM! 
+__NOTE:__ DO NOT USE THE BUILT IN METHODS WHEN ASKED TO MAKE THEM!
 
 1. Using Array.prototype add a new method `myForEach` that mimics the behaivor of the built in `forEach`.
 
-2. Use your `myForEach` to add `myMap` to the Array prototype. `myMap` should behave the same as regular `map`. 
+Array.prototype.myForEach = function(callback) {
+ for (let i = 0; i < this.length; i++) {
+  callback(this[i])
+ }
+};
 
-3. Write a 'myFilter` that behave's the same as `filter`. 
+2. Use your `myForEach` to add `myMap` to the Array prototype. `myMap` should behave the same as regular `map`.
 
-4. Write a `myEvery`. 
+Array.prototype.myMap = function(callback){
+  let arr =[];
+ this.myForEach((el) => {
+  arr.push(callback(el));
+});
+return arr
+};
 
-5. Write a `myReduce`. It should use the first element in the array if none is given. 
+3. Write a 'myFilter` that behave's the same as `filter`.
+
+Array.prototype.myFilter=function(callback){
+  let filteredArr = [];
+  for(let i=0;i<this.length;i++){
+    if (callback(this[i])===true){
+      filteredArr.push(this[i])
+    }
+  }return filteredArr
+}
+
+4. Write a `myEvery`.
+
+Array.prototype.myEvery=function(callback){
+  for(let i=0;i<this.length;i++){
+    if (callback(this[i])!==true){
+      return false
+    }
+  } return true
+}
+
+5. Write a `myReduce`. It should use the first element in the array if none is given.
+
+// unable to answer
```

Comment:
Okay - I'd love to have seen an attempt here. If you write something, even if you know it's wrong, I can look at your code and potentially help. OK?